src/libraries/Microsoft.VisualBasic.Core/tests/Microsoft.VisualBasic.Core.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetCoreAppCurrent);net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);net48-windows</TargetFrameworks>
```

Comment:
Why net48 instead of net461?

Reply:
This test assembly depends on System.ValueTuple which I could have brought in via the `System.Runtime` package for .NET Framework but as the test assembly doesn't reference the source assembly (as it only targets NetCoreAppCurrent) and as we already have other .NET Framework test assemblies which target net48, it felt right to upgrade to being able to access System.ValueTuple inbox.

You might be wondering why I'm doing this now and the reason for that is that before, the test assembly incorrectly referenced the `netstandard2.0` asset of System.Text.Encoding.CodePages instead of the `net461` asset: https://github.com/dotnet/runtime/blob/c94b75f662030a09c8747d79a16d9f4f2c9d44f1/src/libraries/Microsoft.VisualBasic.Core/tests/Microsoft.VisualBasic.Core.Tests.csproj#L56.